generators/vsix-vs-win/BotBuilderVSIX-V4/ProjectTemplates/Multi-project/EchoBot/README.md

```diff
@@ -1,57 +1,65 @@
-﻿This sample demonstrates a simple echo bot with state with ASP.Net Core 2. The bot maintains a simple counter that increases with each message from the user.
-
-# To try this sample
-- Clone the samples repository
-```bash
-git clone https://github.com/Microsoft/botbuilder-samples.git
-```
-- [Optional] Update the `appsettings.json` file under `botbuilder-samples\samples\csharp_dotnetcore\02.echo-with-counter` with your botFileSecret.  For Azure Bot Service bots, you can find the botFileSecret under application settings.
-# Prerequisites
-## Visual Studio
-- Navigate to the samples folder (`botbuilder-samples\samples\csharp_dotnetcore\02.echo-with-counter`) and open EchoBotWithCounter.csproj in Visual Studio.
-- Hit F5.
+﻿# $safeprojectname$
+
+This bot has been created using [Microsoft Bot Framework][1], it shows how to create a simple bot that accepts input from the user and echoes it back.
+
+# To run the bot
 
-## Visual Studio Code
-- Open `botbuilder-samples\samples\csharp_dotnetcore\02.echo-with-counter` sample folder.
-- Bring up a terminal, navigate to `botbuilder-samples\samples\csharp_dotnetcore\02.echo-with-counter` folder.
-- Type 'dotnet run'.
+## Visual Studio
+* Navigate to the folder containing the `.csproj` file and open it in Visual Studio.
+* Run the project (press `F5` key)
 
-## Testing the bot using Bot Framework Emulator
-[Microsoft Bot Framework Emulator](https://github.com/microsoft/botframework-emulator) is a desktop application that allows bot 
-developers to test and debug their bots on localhost or running remotely through a tunnel.
-- Install the Bot Framework emulator from [here](https://aka.ms/botframeworkemulator).
+## .NET Core CLI
+* Install the [.NET Core CLI tools][2].
+* Using the command line, navigate to your project's root folder.
+* Type `dotnet run`.
 
-## Connect to bot using Bot Framework Emulator **V4**
-- Launch the Bot Framework Emulator.
-- File -> Open bot and navigate to `botbuilder-samples\samples\csharp_dotnetcore\02.echo-with-counter` folder.
-- Select `BotConfiguration.bot` file.
-# Bot state
-A key to good bot design is to track the context of a conversation, so that your bot remembers things like the answers to previous questions. Depending on what your bot is used for, you may even need to keep track of conversation state or store user related information for longer than the lifetime of one given conversation.
+## Interacting With Your Bot Using the Emulator
+Launch the [Microsoft Bot Framework Emulator v4][3] and open the generated project's `.bot` file.
```

Comment:
Please use the name of the actual .bot file (It's renamed during the template create)